go/pkg/pwapi/api_season-list.go

```diff
@@ -0,0 +1,14 @@
+package pwapi
+
+import (
+	"context"
+
+	"pathwar.land/pathwar/v2/go/pkg/pwdb"
+)
+
+func (svc *service) SeasonList(ctx context.Context, in *SeasonList_Input) (*SeasonList_Output, error) {
+	var ret SeasonList_Output
+	svc.db.Where(pwdb.Season{IsGlobal: true, IsTesting: false, Status: 1}).Find(&ret.Items)
```

Comment:
checkout this please: https://github.com/pathwar/pathwar/blob/master/go/pkg/pwapi/api_user-get-session.go#L56

Reply:
Thanks, i implemented this function for SeasonList